src/containers/words/SearchWords.js

```diff
@@ -0,0 +1,58 @@
+import { connect } from 'react-redux';
+import React, { PureComponent } from 'react';
+import { withRouter } from 'react-router-dom';
+import qs from 'querystring';
+import SearchForm from '../../components/words/SearchForm';
+import { updateSearchTerm, updateColor } from '../../actions/words';
+import { getSearchTerm, getColor } from '../../selectors/words';
+import PropTypes from 'prop-types';
+
+class SearchWords extends PureComponent {
+  static propTypes = {
+    term: PropTypes.string.isRequired,
+    location: PropTypes.object.isRequired,
+    history: PropTypes.object.isRequired,
+    updateSearchTerm: PropTypes.func.isRequired
+  };
+
+  componentDidMount() {
+    const { term } = qs.parse(this.props.location.search.slice(1));
```

Comment:
```suggestion
    const { term } = qs.parse(this.props.location.search.slice(1));
    if(!term) return;
```
Recommend adding the guard above, because when the page is first rendered no search term has been entered so term comes back undefined. This gets rid of those pesky error messages.